cmd/plume/upload.go

```diff
@@ -126,17 +200,20 @@ func getImageVersion(imagePath string) string {
 	return version
 }
 
-func writeFile(client *http.Client, filename string) error {
+// Write file to Google Storage
+func writeFile(client *http.Client, filename, destname, projec, projectID string) error {
 	ctx := cloud.NewContext(projectID, client)
```

Comment:
So it gets embedded in the context object but nothing in the storage api uses it since cloud/storage doesn't implement creating buckets. I'd say re-use the gce project name here or use an empty string or whatever dummy value. No point in passing around unused data, could lead to surprises as things change over time.